connector/src/main/scala/com/microsoft/kusto/spark/utils/ContainerProvider.scala

```diff
@@ -75,17 +75,20 @@ class ContainerProvider(val client: ExtendedKustoClient, val clusterAlias: Strin
           storageUris(roundRobinIdx)
       }
     } else {
-      Try(client.ingestClient.getResourceManager.getShuffledContainers) match {
```

Comment:
Lets not throw RuntimeException - Also to me this looks like a bug in our Java SDK - because this means we returned here without error and got empty resources